screens/ProfileScreen/ProfileScreen.tsx

```diff
@@ -0,0 +1,183 @@
+        events={user.events}
+      />
+      <ScrollView>
+        <View style={[styles.spacing, styles.imageSection]}>
+          <ProfileImage
+            style={styles.profileImage}
+            imageURL={user.profileImageURL}
+          />
+          <Title style={styles.title}>{user.username}</Title>
+          <Caption>{user.handle}</Caption>
+        </View>
+
+        {userStatus !== "current-user" && (
+          <View style={[styles.spacing, styles.buttons]}>
+            {userStatus === "not-friends" && (
+              <PrimaryButton
+                style={{ flex: 1 }}
+                title="Add Friend"
+                onPress={setStatus}
+              />
+            )}
+            {userStatus === "friend-request-pending" && (
+              <OutlinedButton
+                style={styles.disabledButton}
+                textStyle={styles.textButton}
+                title="Request Pending"
+                disabled
+              />
+            )}
+            {userStatus === "friends" && (
+              <OutlinedButton
+                style={styles.disabledButton}
+                textStyle={styles.textButton}
+                title="Friends"
+                disabled
+              />
+            )}
+            {userStatus !== "blocked" && (
+              <OutlinedButton
+                style={styles.messageButton}
+                title="Message"
+              />
+            )}
+            <ToastWithIcon
+              requestSent={requestSent}
+              setRequestSent={setRequestSent}
+              isVisible={userStatus === "friend-request-pending"}
+              text="Friend request sent."
+            />
+          </View>
+        )}
+
+        <View style={[styles.spacing, { marginVertical: 24 }]}>
+          <Headline style={{ marginBottom: 16 }}>About</Headline>
+          <ExpandableText
+            text={user.biography}
+            linesToDisplay={3}
+            style={{ color: colorText }}
+          />
+        </View>
+
+        <View style={[styles.spacing, { flexDirection: "row"}]}>
+          <Headline>Events</Headline>
+          {user.events.length > 1 && (
+            <View
+              style={{
+                flex: 1,
+                flexDirection: "row",
+                justifyContent: "flex-end"
+              }}
+            >
+              <TouchableOpacity onPress={openModal}>
+                <Headline style={{ color: colorText }}>View All</Headline>
+              </TouchableOpacity>
+            </View>
+          )}
+        </View>
+        {user.events.length !== 0
+          ? (
+            <EventPager events={firstEvents} />
+          )
+          : (
+            <View
+              style={[
+                styles.spacing,
+                {
+                flex: 1,
+                alignItems: "center",
+                justifyContent: "center"
+              }]}
+            >
+              <Caption>Sorry, this user hasn't attended any events</Caption>
```

Comment:
Wonder if we should make this message a little friendlier lol. Take a picture of this and post in Slack